### PR TITLE
Update ICO_Template.cs

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

File: ICO_Template/ICO_Template.cs [hunk 1]
```diff
@@ -145,9 +145,11 @@ public static bool Transfer(byte[] from, byte[] to, BigInteger value)
         {
             if (value <= 0) return false;
             if (!Runtime.CheckWitness(from)) return false;
-            if (from == to) return true;
+            if (to.Length == 20) return false;
```
Comment (erikzhang) [inline ICO_Template/ICO_Template.cs]: `==` or `!=`?

Comment (shargon) [reply ICO_Template/ICO_Template.cs]: != xD

**Comment by shargon** [comment]
I have no rights, we can merge? @erikzhang

**Comment by erikzhang** [comment]
@shargon Can you review https://github.com/neo-project/proposals/pull/44

**Pull request merged**